--- virtuoso-opensource-7.2.16/configure.ac~	2025-11-23 22:16:02.000000000 +0100
+++ virtuoso-opensource-7.2.16/configure.ac	2025-11-23 22:16:45.555077503 +0100
@@ -917,7 +917,7 @@ then
        #elif OPENSSL_VERSION_NUMBER < 0x30600000L
        /* OpenSSL versions 3.0.x - 3.5.x are supported */
 	#else
-	#error OpenSSL version too new
+	//#error OpenSSL version too new
 	#endif
 	    ],[
              AC_MSG_RESULT([$openssl_VERSION is supported])
